boring-sys/build.rs

```diff
@@ -191,4 +194,61 @@ fn main() {
     if cfg!(target_os = "macos") {
         println!("cargo:rustc-cdylib-link-arg=-Wl,-undefined,dynamic_lookup");
     }
+
+    let include_path = PathBuf::from("deps/boringssl/src/include");
+    let mut builder = bindgen::Builder::default()
+        .derive_copy(true)
+        .derive_debug(true)
+        .derive_default(true)
+        .derive_eq(true)
+        .default_enum_style(bindgen::EnumVariation::NewType { is_bitfield: false })
+        .default_macro_constant_type(bindgen::MacroTypeVariation::Signed)
+        .generate_comments(true)
+        .fit_macro_constants(false)
+        .size_t_is_usize(true)
```

Comment:
Even though it's true for the most modern systems, not sure it's always a safe assumption.

Reply:
This is true on all platforms Rust currently works on and Rust doesn't have a `size_t` type.

That recent change in `bindgen` is still being [subject to discussion](https://github.com/rust-lang/rust-bindgen/issues/1901). The previous behavior remains the most intuitive and actually safest way for bindings to interact with C types.